pkg/patch/patch.go

```diff
@@ -144,20 +151,56 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
 	dockerConfig := config.LoadDefaultConfigFile(os.Stderr)
 	attachable := []session.Attachable{authprovider.NewDockerAuthProvider(dockerConfig, nil)}
 	solveOpt := client.SolveOpt{
-		Exports: []client.ExportEntry{
-			{
-				Type: client.ExporterDocker,
-				Attrs: map[string]string{
-					"name": patchedImageName,
-				},
-				Output: func(_ map[string]string) (io.WriteCloser, error) {
-					return pipeW, nil
-				},
-			},
-		},
-		Frontend: "",         // i.e. we are passing in the llb.Definition directly
-		Session:  attachable, // used for authprovider, sshagentprovider and secretprovider
+		Exports:  []client.ExportEntry{},
+		Frontend: "",
+		Session:  attachable,
 	}
+
+	if outputFormat == "docker" {
+        solveOpt.Exports = append(solveOpt.Exports, client.ExportEntry{
+            Type: client.ExporterDocker,
+            Attrs: map[string]string{
```

Comment:
For oci, would it be possible to do this similarly using `client.ExporterOCI`

Reply:
Once it is working for OCI output it would also be good to add an integration test for this